tests/unit/plugins/filter/test_find_template_roles.py

```
@@ -0,0 +1,288 @@
+    TemplateParameter,
+    TemplateParameterDefinition,
+    TemplateTypeEnum,
+    TypeMapping,
+)
+
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+def _make_definition(**overrides) -> TemplateParameterDefinition:
```

Ruff (0.15.21)

[warning] Missing type annotation for `**overrides`

(ANN003)